lib/data/coding_profile_fields.csv

```diff
@@ -1,4 +1,4 @@
 Skills/Languages,text_area,,What tools and languages are you most experienced with? Are you specialized or more of a generalist?,Coding
-I'm getting into,text_area,,What are you learning right now? What are the new tools and languages you're picking up right now?,Coding
-My projects and hacks,text_area,,What projects are currently occupying most of your time?,Coding
+Currently learning,text_area,,What are you learning right now? What are the new tools and languages you're picking up right now?,Coding
```

Comment:
Two more fields that got their label renamed to better reflect the stored attribute name.